release/nightly_tests/setup_chaos.py

```diff
@@ -7,6 +7,7 @@
 def parse_script_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--node-kill-interval", type=int, default=60)
+    parser.add_argument("--max-nodes-to-kill", type=int, default=2)
```

Comment:
2 seems pretty small for the nightly test. why don't we set it like 8?

Reply:
I assume you set default value to 2 for a reason

Reply:
oh I didn't add this argument actually. I guess @stephanie-wang set this 2 to make it more reliable within the CI test environment? Can you confirm?